app/App.js

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
-import { createAppContainer } from 'react-navigation';
+import { createAppContainer, createSwitchNavigator } from 'react-navigation';
 import { createStackNavigator } from 'react-navigation-stack';
 import { createDrawerNavigator } from 'react-navigation-drawer';
 import CourseScreen from './src/views/Course';
+import Login from './src/views/Login';
+// import GoogleLogin from './src/views/GoogleLogin';
```

Comment:
Do we need to keep this in here or can we get rid of it?

Reply:
no we dont